src/GraphControl/GraphControl.vcxproj

```diff
@@ -148,6 +148,7 @@
       <IgnoreAllDefaultLibraries>false</IgnoreAllDefaultLibraries>
       <AdditionalLibraryDirectories>$(GraphingImplLibDir);%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
       <AdditionalDependencies>$(GraphingImplLib);WindowsApp.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <DelayLoadDLLs>GraphingImpl.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
```

Comment:
Do you see any performance gain?

GraphControl is loaded only when the calculator displays the Graphing mode, requiring GraphingImpl.dll to be loaded too.

Reply:
The performance gain is minimal, I think it may be more noticeable in slower machines, but I don't think its true that it only loads when in graphing mode. I've seen it loaded when graphing mode has not yet been visited. I have also confirmed by looking at the modules window in VS while debbugging. I tried adding the GraphControl DLL to the delay loaded dlls for the main Calculator project but it wasn't working for some reason, probably because we use the viewmodels. But this prevents both the bulk of the GraphingImpl dll and internal Graphing Engine dll from loading when a user launches into other modes.